apps/ledger-live-desktop/tests/page/modal/receive.modal.ts

```diff
@@ -36,4 +37,20 @@ export class ReceiveModal extends Modal {
   async expectApproveLabel() {
     await expect(this.approveLabel).toBeVisible();
   }
+
+  @step("Verify send currency / tokens warning message $1")
+  async verifySendCurrencyTokensWarningMessage(account: Account, specificTokens: string) {
+    await expect(this.warningMessage).toBeVisible();
+    await expect(this.warningMessage)
+      .toContainText(`Please only send ${account.currency.ticker} or ${specificTokens} tokens to ${account.currency.name} accounts. 
```

Comment:
Storing text in constants is generally considered a better practice than hardcoding strings directly in the logic